README.md

```diff
@@ -310,74 +333,90 @@ The example requires two trading accounts: `buyer` and `seller`:
+    $ provenanced tx wasm store "ats_smart_contract.wasm" \
         --from "$NODE0" \
+        --node "$NODE" \
         --home "$PIO_NODE" \
         --chain-id "$CHAIN_ID" \
         --keyring-backend test \
         --gas auto \
         --gas-prices 1905nhash \
-        --gas-adjustment 2 \
+        --gas-adjustment 1.2 \
         --testnet \
-        --yes)
-    $ export CODE_ID=$(jq '.logs[0].events[] | select(.type == "store_code").attributes[] | select(.key == "code_id").value | tonumber' <<< "$store_result")
+        --yes \
+        --output json | jq
     ```
+   Find code_id in the result and set
+   ```bash
+   $ export CODE_ID=<<code_id>>
+   ```
 
 3. Instantiate the contract, binding the name `ats-ex.sc.pb` to the contract address:
 
+   _NOTE: It is assumed that the parent name `sc.pb` already exists and is unrestricted._
+
+   <details>
+     <summary>If the name `sc.pb` does not exist, it can be created like so:</summary>
+   
+      ```bash
+      $ provenanced tx name bind "sc" "$NODE0" "pb" \
+          --unrestrict \
+          --from "$NODE0" \
+          --node "$NODE" \
+          --keyring-backend test \
+          --home "$PIO_NODE" \
+          --chain-id "$CHAIN_ID" \
+          --gas-prices 1905nhash \
+          --gas-adjustment 2 \
+          --testnet \
+          --yes
+      ```
+   </details>
+
     ```bash
     $ provenanced tx wasm instantiate "$CODE_ID" \
        '{"name":"ats-ex", "bind_name":"ats-ex.sc.pb", "base_denom":"gme.local", "convertible_base_denoms":[], "supported_quote_denoms":["usd.local"], "approvers":[], "executors":["'$NODE0'"], "ask_required_attributes":[], "bid_required_attributes":[], "price_precision": "0", "size_increment": "1"}' \
         --admin "$NODE0" \
         --label "ats-ex" \
         --from "$NODE0" \
+        --node "$NODE" \
         --home "$PIO_NODE" \
         --chain-id "$CHAIN_ID" \
         --keyring-backend test \
         --gas auto \
         --gas-prices 1905nhash \
-        --gas-adjustment 2 \
+        --gas-adjustment 1.2 \
         --testnet \
         --yes
     ```
 
-   _NOTE: It is assumed that the parent name `sc.pb` already exists and is unrestricted._
-
-   <details>
-     <summary>If the name `sc.pb` does not exist, it can be created like so:</summary>
-
-   ```bash
-   $ provenanced tx name bind "sc" "$NODE0" "pb" \
-       --unrestrict \
-       --from "$NODE0" \
-       --keyring-backend test \
-       --home "$PIO_NODE" \
-       --chain-id "$CHAIN_ID" \
-       --gas-prices 1905nhash \
-       --gas-adjustment 2 \
-       --testnet \
-       --yes
-   ```
-   </details>
-
 4. Get the address of the instantiated contract:
 
    ```bash
-   $ CONTRACT_ADDRESS=$(provenanced q name resolve "ats-ex.sc.pb" --chain-id "$CHAIN_ID" --testnet | awk '{print $2}')
+   $ export=CONTRACT_ADDRESS=$(provenanced q name resolve "ats-ex.sc.pb" --chain-id "$CHAIN_ID" --testnet | awk '{print $2}')
```

Comment:
`export CONTRACT_ADDRESS=`?

Reply:
and actually, i think we need to remove this because we arent using bind names anymore, you would have to get the contract address from the output of the CLI `tx wasm instantiate` or by `provenanced query wasm list-contracts-by-code $CODE_ID` and get the last one

Reply:
cool, thank you!